docs/installation/production/preparation.rst

```diff
@@ -66,7 +66,8 @@ Email Support
 
 If you plan to enable the email capabilities to allow users to reset their forgotten passwords, you'll need to identify an email address to use as the "From" field. Users will be able to reply to these emails, so if your organization has a "noreply" email address, that would be ideal.
 
-* **DEFAULT_FROM_EMAIL**: The "FROM" email address and name of sender that will be shown on forgotten password emails (e.g.: <My App Portal noreply@my.domain.com>).
+* **DEFAULT_FROM_EMAIL**: The "FROM" email address of sender that will be shown on forgotten password emails (e.g.: <noreply@my.domain.com>).
```

Comment:
They're not required. The [Django documentation](https://docs.djangoproject.com/en/2.2/ref/settings/#default-from-email) shows the default value as 'webmaster@localhost' without angle brackets. The [Tethys documentation](http://docs.tethysplatform.org/en/stable/installation/production/configuration/advanced/email_config.html) - Step 4, Note - has always specified "with angle brackets" during the installation instructions. I just used that example for consistency. We really should try to get everyone to stop using the angle brackets, but I included a check to remove them if they exist so that this feature doesn't break current installations.

Reply:
I recommend the following wording:
* **DEFAULT_FROM_EMAIL**: The "FROM" email address of the sender that will be shown on forgotten password emails (e.g.: "noreply@my.domain.com" or "<noreply@my.domain.com>").

Reply:
I'm assuming you mean to have one example without angle brackets and one with?

* **DEFAULT_FROM_EMAIL**: The "FROM" email address of the sender that will be shown on forgotten password emails (e.g.: "noreply@my.domain.com" or "\<noreply@my.domain.com\>").

Reply:
Yes. Sorry.